.github/workflows/docker.yml

```diff
@@ -3,7 +3,7 @@ name: Docker test and publish
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Why not 6.x?

Reply:
We only want to be building containers from the newest branch (main), but ghactions will use the action as defined in the repo's default branch (6.x)

Reply:
ahh, gotcha